android/src/fdroid/play/listings/pl-PL/release-notes.txt

```diff
@@ -1,13 +1,13 @@
-• New OpenStreetMap data as of May 3
-• Fix search for interpolated odd/even house numbers
-• Fix zoom buttons setting
-• Faster rendering of subway layer
-• Fix wrong postcodes in Paris
-• Fix missing Wikipedia links
-• Fix car routing via roundabouts
-• Enable TTS for Portuguese/Brazil, Español/México
-• Fix location detection over a network on some devices
-• Change gates and Lyon subway icon
-• Fix US road shields
-• Add camera, interior_decoration, antiques, art, cheese shops
+• GPX import support
+• New OpenStreetMap data as of June 2
+• UTM and MRGS coordinates support (tap on coordinates to change)
+• Improved car and bicycle routing
+• Fixed bug that prevented upload of OSM edits when on battery power
+• Now compass works on devices without gyroscope
+• Fixed missing Wiki descriptions
+• Decreased TTS (Text-To-Speech) voice speed
+• Added Fish Pass
+• Fixed funicular and parcel locker icons
+• Corrected be, eu, en, en-GB, and uk translations
+
```

Comment:
• Import plików GPX
• Nowe dane OpenStreetMap z 02.06
• Wsparcie dla współrzędnych UTM i MGRS (dotknij, aby przełączyć)
• Ulepszono nawigację samochodową i rowerową
• Poprawiono błąd uniemożliwiający wysłanie edycji OSM na baterii
• Teraz kompas działa również na urządzeniach bez żyroskopu
• Poprawiono brakujące opisy Wiki
• Zmniejszono prędkość komunikatów nawigacji
• Dodano przepławkę
• Poprawiono ikony kolei linowej i paczkomatu
• Poprawiono tłumaczenia: be, eu, en, en-GB oraz uk